aquarius/events/processors.py

```diff
@@ -196,10 +213,11 @@ def process(self):
             raise Exception("RBAC permission denied.")
 
         _record = self.make_record(asset)
+
         if _record:
             try:
                 record_str = json.dumps(_record)
-                self._es_instance.write(record_str, did)
+                self._es_instance.update(record_str, did)
```

Comment:
@calina-c finally the update uses `es.index` that creates the object if it doesn't exist and updates it if it does (I think as you mentioned)